src/main/java/net/minecraftforge/fmp/capabilities/ISlottedCapabilityProvider.java

```diff
@@ -0,0 +1,14 @@
+package net.minecraftforge.fmp.capabilities;
+
+import net.minecraft.util.EnumFacing;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.fmp.multipart.PartSlot;
+
+public interface ISlottedCapabilityProvider
+{
+    
+    public boolean hasCapability(Capability<?> capability, PartSlot slot, EnumFacing facing);
+
+    public <T> T getCapability(Capability<T> capability, PartSlot slot, EnumFacing facing);
```

Comment:
It's not meant for things like entities or items, only for things that are slot-aware, like a multipart container. I could throw it in IMultipartContainer if you want.